api-reference/_hidden/GridBaseColumn/encodeHtml.md

```diff
@@ -8,4 +8,6 @@ default: true
 Specifies whether HTML tags are displayed as plain text or applied to the values of the column.
 
 ---
-When **true**, HTML tags are displayed as plain text; when **false**, they are applied to the values of the column.
\ No newline at end of file
+When **true**, HTML tags are displayed as plain text; when **false**, they are applied to the values of the column.
+
+[note]When exporting, this option's value is ignored and formatting is always exported.
```

Comment:
There are two possible variants: tags as plain text and tags applied to the values. Let's express this thought in these terms, without other variants like "formatting is always exported". Also, let's not give this info as a note.

Reply:
'When exporting, tags are always applied to the values'?

Reply:
From what I gather, tags are exported as a plain text.

Reply:
Right you are. I messed this up when I tested export, now I tried again. ''When exporting, tags are always exported as plain text'?